Day 3/solution2/src/main.rs

```diff
@@ -1,68 +1,23 @@
-use std::fs::File;
-use std::io::{self, BufRead};
-use std::path::Path;
-// use std::slice;
+use std::fs;
 
 fn main() {
-    // let mut counts : Vec<Vec<u32>> = vec![vec![0, 2]; 12];
-    let count: Vec<u32> = {
-        let lines: io::Lines<_> = read_lines("../input.txt").unwrap();
-        let lines_vector: Vec<String> = lines.map(|line| line.unwrap()).collect::<Vec<String>>();
-        let lines_count_vector: Vec<Vec<u32>> = lines_vector
```

Comment:
One thing while operating on data just moves along a pipeline in Rust is that you can get away with stringing stuff together quite easily. Sometimes this can be less readable, but more often, it allows you to be more concise while the intention is still there.